hiring_2024.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: Join Us
+permalink: /quantamole/
+---
+
+## About the group
+
+The Polyquantique research group led by Nicolás Quesada specializes in quantum photonics and quantum information. The group is looking for a highly motivated candidates at the *PhD* and *postdoctoral* levels with interest/expertise in quantum optics.
+
+We are a theory group with a significant computational component that collaborates closely with experimental groups and partners in government and industry. We write [open source code](https://open-science-training-handbook.gitbook.io/book/open-science-basics/open-research-software-and-open-source) and do [reproducible science](https://blogs.egu.eu/divisions/gd/2018/09/19/reproducible-computational-science/) (that is, making all data and code available to others to easily execute).  
+
+We believe that a significant part of scientific research can be understood as collective problem solving. Not having a group that can look at problems from diverse vantage points is a wasted opportunity. We are thus committed to building a diverse research group from its conception and to being attentive to and supportive of the needs and ambitions of students and postdocs, especially those who belong to marginalized or underrepresented groups in science and engineering. 
+
+We are based in beautiful and welcoming Montréal, a safe but never boring city that consistently ranks as the best city in North America to be a student.
+
+## About the position
+
+As part of the QuantaMole consortium we are interested in studying novel materials with extremely high nonlinearities and their application as potential mediators of interactions between photons. A significant no-go result in this field is 
+that ["Single-photon Kerr nonlinearities do not help quantum computation"](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.73.062305). We are interested in going beyond this result studying other geometries, configurations and topologies in which new materials can provide the sought-after interactions. This might include resonating geometries, utilizing χ<sup>2</sup> nonlinerities instead of χ<sup>3</sup> and new states of light in which to, more succesfuly, apply these nonlinear operations.
```

Comment:
```suggestion
As part of the QuantaMole consortium, we are interested in studying novel materials with extremely high nonlinearities, as well as their application as potential mediators of interactions between photons. A significant no-go result in this field is 
that ["Single-photon Kerr nonlinearities do not help quantum computation"](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.73.062305). We are interested in going beyond this result and study other geometries, configurations and topologies in which new materials can provide the sought-after interactions. This might include resonating geometries, utilizing χ<sup>2</sup> nonlinearities instead of χ<sup>3</sup>, and new states of light in which to apply these nonlinear operations with more success.
```